docs/src/howto-guides/025_config_management_plugin.md

```diff
@@ -0,0 +1,92 @@
+# How to manage configuration files with Cumulocity
+
+With `thin-edge.io`, you can manage config files on a device by using the [Cumulocity configuration management feature](https://cumulocity.com/guides/users-guide/device-management/#managing-configurations) as a part of Device Management.
+
+If you are new to the Cumulocity **Configuration Management** feature,
+we recommend you to read [the Cumulocity user guide](https://cumulocity.com/guides/users-guide/device-management/#managing-configurations) along with this how-to guide.
+
+## Installation of `c8y_configuration_plugin`
+
+To enable the feature, first you need to install the `c8y_configuration_plugin` binary on your device.
+
+### With debian package
+
+For Debian distribution OS, we provide the `c8y_configuration_plugin_<version>_<arch>.deb` package as a release asset [here](https://github.com/thin-edge/thin-edge.io/releases).
+
+If you have used `get_thin_edge_io.sh` script to install `thin-edge.io`, this package is installed, by default.
+You can skip this section.
+
+In case that you didn't use the script, you can get the package on our [Releases](https://github.com/thin-edge/thin-edge.io/releases) and install it.
+
+### For non-Debian OS distribution
+
+To get the `c8y_configuration_plugin` binary, you can either extract it from the debian package or build from the sources.
+
+#### Extracting from debian package
+
+Get the `c8y_configuration_plugin_<version>_<arch>.deb` from our [Releases](https://github.com/thin-edge/thin-edge.io/releases).
+Then, follow our guide [Extracting from debian package](./015_installation_without_deb_support.md#extracting-binaries-from-deb-packages).
+
+#### Building from sources
+
+Follow our guide [Building from source](./015_installation_without_deb_support.md#if-building-from-source).
+
+A `systemd` unit file for `c8y_configuration_plugin` can be found in the repository at `configuration/init/systemd/c8y-configuration-plugin.service`
+and should be installed on the target in: `/lib/systemd/system/c8y-configuration-plugin.service`.
+
+## Get started
+
+Before starting anything, first finish [establishing the connection to Cumulocity](./../tutorials/connect-c8y.md).
+
+**Step 0**
+Only if you didn't install `c8y_configuration_plugin` by the debian package,
+you need one additional step to initialize the plugin. Run this command.
+
+```shell
+sudo c8y_configuration_plugin --init
+```
+
+**Step 1**
+Modify the `/etc/tedge/c8y/c8y-configuration-plugin.toml` file in the [format written in the specification](./../references/c8y-configuration-management.md#configuration).
```

Comment:
```suggestion
Add the files you want to configure from the cloud into the `/etc/tedge/c8y/c8y-configuration-plugin.toml` file using the [format described in the reference guide](./../references/c8y-configuration-management.md#configuration).
```

I would also:
* Add an example
* Say this file can be let empty and configured from the cloud
* Provide a link to the reference guide as a note after the example and not in the first sentence.

Reply:
Now `c8y_configuration_plugin --init` creates an example configuration file in `/etc/tedge/c8y/c8y-configuration-plugin.toml`. So it doesn't need to be empty.